activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-rest/src/test/java/org/activiti/cloud/services/modeling/rest/controller/ModelValidationControllerIT.java

```diff
@@ -114,7 +114,7 @@ public void should_returnStatusNoContent_when_validatingProcessModelWithValidCon
 
         mockMvc
             .perform(multipart("/v1/models/{model_id}/validate", processModel.getId()).file(file))
-            .andExpect(status().isNoContent());
+            .andExpect(status().isBadRequest());
```

Comment:
The name of the test says `should_returnStatusNoContent_when_validatingProcessModelWithValidContent` while here the expected status is changed to `Bad Request`. The test case is actually testing the positive path and we should rather in case of an implementation change we should rather:

- fix the test case so that it still tests the positive path
- create a new test case for the `Bad Request` path